arevyn24-assignment-4/src/arevyn24/assignment/pkg4/Assignment4Methods.java

```diff
@@ -0,0 +1,329 @@
+/**
+ * @author Samantha Shurie 
+ * 
+ * JAVA I - CPT 163-27:
+ * This program is a modification of assignment 2, where we were to design a
+ * menu application. This program moves each of the previous menu options into
+ * their own methods.
+ */
+
+
+package arevyn24.assignment.pkg4;
+import java.util.Random;    //needed to generate random numbers
+import java.util.Scanner;   //needed for reading user input
+
+public class Assignment4Methods {
+    
+    
+    /**
+     * @param args the command line arguments
+     */
+    public static void main(String[] args) {
+       Integer input = 0; //User menu selection
+       Scanner keyboard = new Scanner(System.in);
+       
+       
+      //While loop: run this menu as long as the user does not enter 6 on the main menu
+        while (input != 6) {
+            
```

Comment:
It doesn't really effect the operation of the code, but you could have used a Do while loop here since the code should always run at least once. No need to pretest to loop. This works fine though.